circle.yml

```diff
@@ -1,72 +0,0 @@
-# These environment variables must be set in CircleCI UI
```

Comment:
Sorry, I wasn't very explicit. I meant please put this comment back at the top of the file since it still applies. It seems important.

Reply:
Done!